### PR TITLE
Company details

### PR DESCRIPTION
**Pull request by teneightfive**
Add support for basic company details

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 module.exports = {
-  companiesHouseApiKey: process.env.COMPANY_KEY,
-  companySearchUrl: 'https://api.companieshouse.gov.uk/search/companies',
-  officerSearchUrl: 'https://api.companieshouse.gov.uk/search/officers'
+  companiesHouse: {
```
Comment (ztolley) [inline config.js]: totally getting dejavue

File: controllers/CompanyViewController.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const api = require('../lib/companiesHouseApi');
+let companiesData = require('../data/companies.json');
+
+function get(req, res) {
+  let companyNum = req.params.id;
+
```
Comment (ztolley) [inline controllers/CompanyViewController.js]: What if there's no id in the params?

File: views/company.jade [hunk 3]
```diff
@@ -0,0 +1,71 @@
+extends ukti
+
+block content
+  h1= company.company_name
+  .tabs.tabs--inline.js-tabs
```
Comment (ztolley) [inline views/company.jade]: create own fragment

File: lib/companiesHouseApi.js [hunk 4]
```diff
@@ -1,42 +1,43 @@
...
 
-const KEY = config.companiesHouseApiKey;
-const COMPANY_BASE = config.companySearchUrl;
-const OFFICER_BASE = config.officerSearchUrl;
-
-function search(url, query) {
+function search(endpoint, params) {
   const options = {
-    url: url,
-    qs: { q: query },
```
Comment (ztolley) [inline lib/companiesHouseApi.js]: Change to use query instead of q, consistent and more descriptive

Comment (teneightfive) [inline lib/companiesHouseApi.js]: The API needs the query param to be `q` though

File: controllers/CompanyViewController.js (hunk 2 above)
Comment (teneightfive) [inline controllers/CompanyViewController.js]: Is now being handled by redirecting to dashboard.

File: lib/companiesHouseApi.js (hunk 4 above)
Comment (ztolley) [inline lib/companiesHouseApi.js]: Yeah I just meant our side.

**Pull request merged**